cobbler/cobblerd.py

```diff
@@ -32,6 +32,11 @@
 
 
 def core(api):
+    """
+    Starts cobbler.
```

Comment:
Nitpick: I notice both "cobbler" and "Cobbler" are used in various places in all files.